mcs/class/System/Test/System.Net/HttpWebRequestTest.cs

```diff
@@ -567,7 +567,7 @@ public void EndGetRequestStream_AsyncResult_Null ()
 		}
 
 		[Test]
-		[Category ("NotWorking")] // do not get consistent result on MS
```

Comment:
I'm not sure whether we should re-enable this, it has the potential to cause problems.  I will add something to the web-tests once I've done cleaning up some other things.  A correct and reliable way of testing this is to wait until the request has actually started - which requires the web-tests' instrumentation framework.